pkg/autodiscover/autodiscover_podset.go

```diff
@@ -92,10 +91,10 @@ func findStatefulSetByLabel(
 		for _, l := range labels {
 			label := buildLabelQuery(l)
 			logrus.Trace("find StatefulSet in ", ns, " using label ", label)
-			options := metav1.ListOptions{}
-			options.LabelSelector = label
 			statefulSetClient := appClient.StatefulSets(ns)
-			ss, err := statefulSetClient.List(context.TODO(), options)
+			ss, err := statefulSetClient.List(context.TODO(), metav1.ListOptions{
```

Comment:
changes not needed in this file due to  https://github.com/test-network-function/cnf-certification-test/pull/157/files

Reply:
That PR is newer than mine... 😄

Reply:
It was my fault, sorry, I didn't notice this PR. I'll rebase mine.